cmd/kubeadm/app/cmd/join.go

```diff
@@ -72,17 +72,17 @@ func NewCmdJoin(out io.Writer) *cobra.Command {
 	}
 
 	cmd.PersistentFlags().StringVar(&cfgPath, "config", cfgPath, "Path to kubeadm config file")
+	cmd.PersistentFlags().StringVar(&cfg.DiscoveryURL, "discovery-url", "", "Discovery URL")
+	cmd.PersistentFlags().StringVar(&cfg.DiscoveryFile, "discovery-file", "", "Discovery File path")
+	cmd.PersistentFlags().StringVar(&cfg.DiscoveryToken, "discovery-token", "", "Discovery Token")
+	cmd.PersistentFlags().StringVar(&cfg.TLSBootstrapToken, "tls-bootstrap-token", "", "blabla")
```

Comment:
"The token to use when authenticating to the API server for bootstrap TLS credentials"